cloud/google/gce.py

```diff
@@ -141,7 +147,7 @@
 
 requirements:
     - "python >= 2.6"
-    - "apache-libcloud >= 0.13.3"
+    - "apache-libcloud >= 0.17.0"
```

Comment:
Does this actually change the Ansible installation process? I was under the impression that this was just guidance since the GCE guide had [users installing libcloud](http://docs.ansible.com/ansible/guide_gce.html#introduction) outside of the Ansible process.